<launch>

    <!-- smach node -->
    <node name="smach_node" pkg="behavioral_model" type="model_2012.py" output="screen"/>
    <node name="smach_viewer" pkg="smach_viewer" type="smach_viewer.py" output="screen" />

</launch>
